__tests__/news-server.test.js

```diff
@@ -0,0 +1,41 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
You must test the length of the array because if the length was 0 the tests in the forEach would never run